test/link/test.sh

```diff
@@ -110,27 +121,45 @@ for i in fragment-align/*; do
 	fi
 done
 
+i="high-low.asm"
+startTest
 $RGBASM -o $otemp high-low/a.asm
 rgblink -o $gbtemp $otemp
 $RGBASM -o $otemp high-low/b.asm
 rgblink -o $gbtemp2 $otemp
-i="high-low.asm" tryCmp $gbtemp $gbtemp2
+tryCmp $gbtemp $gbtemp2
+rc=$(($? || $rc))
+
+i="overlay.asm"
+startTest
+$RGBASM -o $otemp overlay/a.asm
+rgblink -o $gbtemp -t -O overlay/overlay.gb $otemp > $outtemp 2>&1
+tryDiff overlay/out.err $outtemp
+rc=$(($? || $rc))
+tryCmp overlay/out.gb $gbtemp
```

Comment:
I would just add a comment pointing out that this test also intentionally checks the output's size.